Assets/Scripts/AvatarLoadingHandler.cs

```diff
@@ -1,26 +1,29 @@
+
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine;
 using UnityEngine.Serialization;
 using Random = UnityEngine.Random;
+using ReadyPlayerMe.Core;
 
 namespace ReadyPlayerMe.Loadtest
 {
     public class AvatarLoadingHandler : MonoBehaviour
     {
-        [SerializeField] private string baseUrl = "https://api.readyplayer.me/v1/avatars/";
+        [SerializeField] private string baseUrl = "https://models.readyplayer.me/";
```

Comment:
not really an issue but internally we use this as base URL https://api.readyplayer.me/v1/avatars/
![image](https://github.com/readyplayerme/Unity-Loadtest/assets/7085672/f783cbcc-a964-4fe4-a505-c4f668aaee26)
Part of the reasoning was that initially models URL was only going to be temporary, it was just required to fix an issue in SDK with URL regex/validation. 
Not sure if we still plan to drop support for it though.

Reply:
i guess as it is part of a loadtester, we should aim for best performance - hence using the cdn infront.